charts/metrics-server/README.md

```diff
@@ -22,57 +22,58 @@ helm upgrade --install metrics-server metrics-server/metrics-server
 
 The following table lists the configurable parameters of the _Metrics Server_ chart and their default values.
 
-| Parameter                            | Description                                                                                                                                                                                                                                                      | Default                                         |
```

Comment:
/cc @stevehipwell 
why do we modify this again? Isn't this pr already formatted?
https://github.com/kubernetes-sigs/metrics-server/pull/1107/files

Reply:
Since I added a value to values.yaml that I can set, I need to add this value to the README table

Reply:
@yangjunmyfm192085 it's because the table width is defined by the content and this is the standard behaviour of Markdown formatters. IMHO formatters are good but it'd be better if we could use an alternative structure for tables to avoid this churn.